package.json

```diff
@@ -57,7 +57,6 @@
     "postcss-cli": "*",
     "autoprefixer": "*",
     "browser-sync": "*",
-    "devtools-terminal": "*",
```

Comment:
@rianby64 Also have updated some dependencies due to security risk. Check for merging of #181 which may cause (easy) conflict that may need fixing. But DEFINITELY want to get rid of this.

Reply:
Bad news. If you rely on "devtools-terminal" then please, check the following:
https://github.com/chjj/pty.js#todo
> Add a way of determining the current foreground job for platforms other than Linux and OSX/Darwin.

`devtools-terminal` relies on `pty` which works only on Mac.

Reply:
@rianby64 EXACTLY. Let's kill it with fire. To be honest it only was around when I was testing VIM in browser.  I'm going to go ahead and merge this.